docs/integration_quality_scale_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Integration Quality Scale"
+sidebar_label: "Introduction"
+---
+
+The Integration Quality Scale scores each integration based on the code quality and user experience. Each level of the quality scale consists of a list of requirements. If an integration matches all requirements, it's considered that level.
+
+> Suggestions for changes can be done by creating an issue in the [architecture repo](https://github.com/home-assistant/architecture/issues/).
+
+# No score
+
+This integration passes the bare minimum requirements to become part of the index.
+
+ - Satisfy all requirements for [creating components](creating_component_code_review.md) and [creating platforms](creating_platform_code_review.md).
+ - Configurable via `configuration.yaml`
+
+# Silver 🥈
+
+This integration is able to cope when things go wrong. It will not print any exceptions not will it fill the log with retry attempts.
+
+- Set an appropriate `SCAN_INTERVAL` (if a polling integration)
+- Raise `PlatformNotReady` if unable to connect during setup (if appropriate)
+- Handles expiration of auth credentials. Refresh if possible or print correct error and fail setup. If based on a config entry, should trigger a new config entry flow to re-authorize.
+- Handles internet unavailable. Log a warning once when unavailable, log once when reconnected.
+- Handles device/service unavailable. Log a warning once when unavailable, log once when reconnected.
+- Set unavailable property to False if appropriate ([docs](entity_index.md#generic-properties))
```

Comment:
I think this is meant to say
Set `available` property to False when appropriate